cmd/notary-server/config.json

```diff
@@ -11,5 +11,8 @@
 	},
 	"logging": {
 		"level": 5
+	},
+	"storage": {
+		"db_url": "dockercondemo:dockercondemo@tcp(notarymysql:3306)/dockercondemo"
```

Comment:
Is this a dev-config? if so, shouldn't the file reflect it? Also, we should start deleting mentions to dockercon.